liferay-workspace/tests/build.gradle

```diff
@@ -0,0 +1,14 @@
+apply plugin: 'groovy'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    testCompile 'org.codehaus.groovy:groovy-all:2.4.4'
+    testCompile 'org.spockframework:spock-core:1.0-groovy-2.4'
+	testCompile gradleTestKit()
+}
+
+check.dependsOn ':blade-tests:initBundle'
```

Comment:
Where is the `:blade-tests` coming from?  There is no project called that right?